ipc/metrics/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ipc_metrics"
```

Comment:
I don't think this has to be a crate. IMO it should live under `ipc_cli` which hosts them.